tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -51,7 +51,7 @@ BenchmarkProblems:
           - [16,16,1]
         - GlobalReadVectorWidthA: [2,4,8]
         - GlobalReadVectorWidthB: [4,8]
```

Comment:
I don't see any test cases that is testing DTVA+TT+Tail and DTVB+NN+Tail ? Perhaps you can go to the third ticket and download the 2 yamls, and put a few cases.

i.e. a block that has **DTVA + TT + Tailed-size and non-tailed-size**, and a block that has **DTVB + NN + Tailed-size vs non-tailed-size**. You can directly see the attached yamls in that ticket.